### ISSUE TITLE
How to improve our model? finetuning

### ISSUE THREAD

**Issue by EmbraceLife** [EmbraceLife/shendusuipian#100]
### How to improve our model? finetuning
74:37 - 76:26
[How to improve our model](https://ytcropper.com/cropped/Xf5c6c848ee9bd2)
What is the default way of training?
	- add a few layers at the end
	- only train or update weights for the last few layers
What is the benefit of the default way?
	- less likely to overfit
	- much faster
How to train the whole model?
	- unfreeze the model `learn.unfreeze()`
	- train the entire model `learn.fit_one_cycle(1)`
why it is easy to ruin the model by training the whole model?
	- learning rate is more likely to set too large for earlier layers *
	- to understand it please see the next question
目录 Lesson 1 知识点分解
